Dangerfile

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env ruby
+
+require 'active_support/core_ext/string/output_safety'
+
+def verifyPRTitle()
+    title = github.pr_title
+    if title.size < 17
+        warn("Title too short. It should fit '[KLTB002-JIRA_TICKET_NO] TITLE' template.")
```

Comment:
Can we use a configured value for the project code, instead of hardcoding it in the Dangerfile ?
Maybe something like this https://github.com/danger/swift/issues/213 ?

Reply:
Check comment above.
Yep, good idea, let's see if it works.